src/api/goods.ts

```diff
@@ -3,17 +3,48 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
+const request = async (url: string) => {
```

Comment:
will be better to add types for returned value

Reply:
I will fix it